src/cgame/cg_local.h

```diff
@@ -909,16 +913,14 @@ typedef struct
 	int         legBones[ MAX_BONES ];
 	int         numLegBones;
 
-	int         weaponAdjusted; // bitmask of all weapons that have hand deltas
-	int         handBones[ MAX_BONES ];
-	int         numHandBones;
-
 	sfxHandle_t customFootsteps[ 4 ];
 	sfxHandle_t customMetalFootsteps[ 4 ];
 
 	char        voice[ MAX_VOICE_NAME_LEN ];
 	int         voiceTime;
-} clientInfo_t;
+	std::vector<std::shared_ptr<SkeletonModifier>> modifiers;
```

Comment:
We need to store pointers here so that we can share one copy of a modifier among all the different clientInfos. There is one per player model and they are generally shared.